packages/design-system/src/components/OcSelect/OcSelect.vue

```diff
@@ -24,6 +24,35 @@
         <oc-spinner v-if="loading" />
       </template>
     </vue-select>
+
+    <div
+      v-if="showMessageLine"
+      class="oc-text-input-message"
+      :class="{
+        'oc-text-input-description': !!descriptionMessage,
+        'oc-text-input-warning': !!warningMessage,
+        'oc-text-input-danger': !!errorMessage
+      }"
+    >
+      <OcIcon
+        v-if="messageText !== null && !!descriptionMessage"
+        name="information"
+        size="small"
+        fill-type="line"
+        accessible-label="info"
+        aria-hidden="true"
```

Comment:
No need to add the `aria-hidden` attribute. OcIcon handles that automatically if no accessible label is provided (see comment above).